src/20_Components/amp-experiment.html

```diff
@@ -0,0 +1,80 @@
+<!---{
+    "experiment": true,
+    "component": "amp-experiment"
+  }--->
+<!-- 
+  #### Introduction
+
+  [amp-experiment](https://github.com/ampproject/amphtml/blob/master/extensions/amp-experiment/amp-experiment.md) allows to perform user experience experiments on an AMP document and collect resulting data.
```

Comment:
We should use both phrases. A/B testing is what most people will be searching for - so we should include the term.